projects/packages/videopress/src/client/state/reducers.js

```diff
@@ -245,8 +246,10 @@ const videos = ( state, action ) => {
 
 			return {
 				...state,
+				removedVideos: [ id, ...( state.removedVideos ?? [] ) ],
+				removedVideoCount: ( state.removedVideoCount ?? 0 ) + 1,
```

Comment:
Not a big thing, but `removed` looks strange to me since we're looking for videos that are in the process of being removed.

Maybe a `removing` instead `removed`?